frontend/packages/dev-console/src/components/pipelineruns/PipelineRunDetailsPage.tsx

```diff
@@ -24,6 +25,11 @@ const PipelineRunDetailsPage: React.FC<DetailsPageProps> = (props) => {
       pages={[
         navFactory.details(PipelineRunDetails),
         navFactory.editYaml(viewYamlComponent),
+        {
+          href: 'TaskRuns',
```

Comment:
Please change this to lower case hyphenated words in the url, so that we are using similar url segments in both the perspectives.
```suggestion
          href: 'task-runs',
```

Reply:
Odd, the Pipeline Details Page has a "Runs" Tab for Pipeline Runs 😕 

Definitely agree we should `kebab-case` the route names though.